shoov/drupal-org.make

```diff
@@ -95,13 +95,13 @@ projects[restful][subdir] = "contrib"
 projects[restful][type] = module
 projects[restful][download][type] = git
 projects[restful][download][branch] = 7.x-1.x
-projects[restful][download][revision] = 9dae54204eef0925081abcfaaa20faa6191a3a4c
+projects[restful][download][revision] = a12e801c25df411752748e56fbb94bae7a7968be
```

Comment:
I thought since we used commit before - we're using dev version. So I've took the last commit. Change to 7.x-1.6?